src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -17,5 +19,8 @@ public SelectTestFigureOptionListener(DriverManager driverManager) {
 	@Override
 	public void actionPerformed(ActionEvent e) {
 		FiguresJoe.figureScript1(driverManager.getCurrentDriver());
+		DriverCommandManager manager = CommandsFeature.getDriverCommandManager();
+		FiguresJoe.figureScript1(manager);
+		manager.setCurrentCommand(manager.getCommands(),"FigureJoe1");
```

Comment:
You should not modify those tests. It should be not necessary to test the feature.